cardinal_toe_app/src/app/app.component.scss

```diff
@@ -5,36 +5,46 @@
     box-sizing: border-box;
     display: grid;
     grid-template: 
-        "header header" 60px
-        "first-panel second-panel" auto
-        "footer footer" 60px / 15% auto;
-    background-color: burlywood;
+        "header header" 80px
+        "first-panel second-panel" auto / 15% auto;
+        // "footer footer" 60px / 15% auto; 
```

Comment:
you shouldn't really commit commented out code

Reply:
all footer code commented out in case we find a use for it in the future.